salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -4,9 +4,40 @@
 {%- set htpasswd_path = "/etc/kubernetes/htpasswd" %}
 {%- set encryption_k8s_path = "/etc/kubernetes/encryption.conf" %}
 
+{# This whole block is used to "know" the Ingress external IP used by Dex.
```

Comment:
I split that into a macro, why put it back here? It's a huge block of Jinja code that's not specific to API Server, hindering readability... But not critical, just questioning.

Reply:
Yeah, I can add this split back.

The split was done in your PR that adds `Nginx certs`
Reviewer requested it to removed since it was not used.